### DISCUSSION TITLE
add_raster toggle raster longitudonal continuity

### DISCUSSION THREAD

**Discussion by aluhamaa** [opengeos/leafmap#941] [Q&A] [answered]
Hi
Currently, when I plot a raster on the map, it is continuous in longitude, as shown in the image. This is fine in most cases, but sometimes I would like to see only one version, ranging from -180 to 180, for example. To speed up processing, etc. Is there an easy way to achieve this behavior?

<img width="967" alt="Screenshot 2024-10-29 at 16 01 52" src="https://github.com/user-attachments/assets/1ce88bf1-48ea-448f-a901-7ee74059b8d0">

I'd like to optionally use the version I get with displaying polygons, which is just one version and not repeated:

<img width="1028" alt="Screenshot 2024-10-29 at 16 01 40" src="https://github.com/user-attachments/assets/30200278-6019-4ea8-b45e-fb1f39b9b481">

**Comment by giswqs** [comment, marked as answer]
This is an issue for ipyleaflet than leafmap. The ipyleaflet Map class has the `world_copy_jump` attribute, which is supposed to work in this case. It seems not. You may want to open an issue at https://github.com/jupyter-widgets/ipyleaflet/issues. 
`Map(world_copy_jump=False)`
https://ipyleaflet.readthedocs.io/en/latest/map_and_basemaps/map.html#ipyleaflet.Map.world_copy_jump
